### ISSUE TITLE
Perform user research

### ISSUE THREAD

**Issue by matt-dray** [co-analysis/a11ytables#50]
I've been trying relatively quickly to get the minimum viable product. I don't—and _you_ shouldn't—trust my design decisions and opinions. 

We have two major user groups to test to make sure the package is fit for purpose:

* citizens/users of the output spreadsheet, i.e. does it meet the accessibility guidance and if not, why not?
* government analysts (maybe wider R users) who use the package itself, i.e. does the API make sense and can the  documentation be followed?

This may well be the job of the ONS Best Practice team if/when this package, or aspects of it, are passed over to them for further development.

Bottom line: actual humans need to try this.

**Comment by matt-dray** [comment]
Actual humans have tried the package and sent me feedback via issues and emails for a while now. Changes have been made and the package has been improved/will be improved on this basis.
